jobtalk/urls.py

```diff
@@ -16,11 +16,15 @@
 from django.conf.urls import url
 from django.contrib import admin
 
-from core.views import UserRegistration, successful, ReviewSubmission
+from core.views import (UserRegistration, successful, ReviewSubmission,
+    ReviewList, ReviewDetail)
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url(r'^users/', UserRegistration.as_view(), name='user-registration'),
-    url(r'^reviews/', ReviewSubmission.as_view(), name='review-submission'),
+    url(r'^reviews$', ReviewList.as_view(), name='review-list'),
+    url(r'^reviews/$', ReviewList.as_view(), name='review-list'),
+    url(r'^reviews/submit$', ReviewSubmission.as_view(), name='review-submission'),
```

Comment:
I think we should follow a url standard. I have seen most of the django dev follows slash at the end of the url. So we should also add `/` at end of the url
I think we can remove the ``/reviews`` from the url list.